app/models/ability.rb

```diff
@@ -148,6 +148,6 @@ def initialize(user, token = nil)
   # rubocop:enable Metrics/AbcSize, Metrics/PerceivedComplexity, Metrics/CyclomaticComplexity, Metrics/MethodLength
 
   def in_scan_pilot_group?(user)
-    user.affiliation.any? { |g| Settings.scan_pilot_groups.include? g }
+    Settings.folio.scan_pilot_groups.include?(user.patron.patron_group_name)
```

Comment:
Don't we need to keep checking the user affiliation for current requests, because we don't know their patron until after they submit the request?

Reply:
In the current system, users don't log in until they submit the request.